### PR TITLE
Changing the app category to `game` in `android`.

### PR DESCRIPTION
**Pull request by MAJigsaw77**
the title is saying all but the pr must be pushed after [this pr](https://github.com/openfl/lime/pull/1724) is pushed.

### CODE REVIEW

**Comment by Geokureli** [comment]
Can you summarize what happened? Why was this closed?

**Comment by MAJigsaw77** [comment]
> Can you summarize what happened? Why was this closed?

Because of this https://github.com/openfl/lime/pull/1735

**Comment by MAJigsaw77** [comment]
It's literally better than my implementation

**Comment by Geokureli** [comment]
> It's literally better than my implementation

Should we add default or commented out tags to flixel's project.xml template?

**Comment by MAJigsaw77** [comment]
> > It's literally better than my implementation
> 
> Should we add default or commented out tags to flixel's project.xml template?

When that pr is pushed you should change the app category to `game` seems this is a game engine.
